Marlin/src/module/temperature.cpp

```diff
@@ -2774,18 +2781,24 @@ void Temperature::isr() {
     #if ENABLED(SHOW_TEMP_ADC_VALUES)
       , const float r
     #endif
-    , const int8_t e=-3
+    , const int8_t e=-4
   ) {
-    #if !(HAS_HEATED_BED && HAS_TEMP_HOTEND && HAS_TEMP_CHAMBER) && HOTENDS <= 1
+    #if !(DISABLED(TEMP_SENSOR_1_AS_REDUNDANT) && HAS_HEATED_BED && HAS_TEMP_HOTEND && HAS_TEMP_CHAMBER) && HOTENDS <= 1
       UNUSED(e);
     #endif
 
     SERIAL_CHAR(' ');
     SERIAL_CHAR(
-      #if HAS_TEMP_CHAMBER && HAS_HEATED_BED && HAS_TEMP_HOTEND
+      #if ENABLED(TEMP_SENSOR_1_AS_REDUNDANT) && HAS_TEMP_CHAMBER && HAS_HEATED_BED && HAS_TEMP_HOTEND
+        e == -3 ? 'R' : e == -2 ? 'C' : e == -1 ? 'B' : 'T'
+      #elif HAS_TEMP_CHAMBER && HAS_HEATED_BED && HAS_TEMP_HOTEND
         e == -2 ? 'C' : e == -1 ? 'B' : 'T'
+      #elif ENABLED(TEMP_SENSOR_1_AS_REDUNDANT) && HAS_HEATED_BED && HAS_TEMP_HOTEND
++        e == -3 ? 'R' : e == -1 ? 'B' : 'T'
       #elif HAS_HEATED_BED && HAS_TEMP_HOTEND
         e == -1 ? 'B' : 'T'
+      #elif ENABLED(TEMP_SENSOR_1_AS_REDUNDANT) &&  HAS_TEMP_HOTEND
++        e == -3 ? 'R' : 'T'
```

Comment:
I'm not sure we can add to the temperature report without disrupting host software. And there is some debate whether the redundant sensor needs to be reported at all outside of a debugging mode.

CC: @repetier @foosel @MarlinFirmware/host-software-team

Reply:
I wondered about that, I have tried with Cura 4.1, pronterface and astroprint; without seeing a problem but we can skip that piece. Or I could add a debug option for redundant sensor.

Reply:
Can't dive into the code right now, how would a temperature report with this added on look like?

Reply:
Here is an example of report as now - temp_0 and bed sensors
 T:190.26 /190.00 B:59.98 /60.00
1st value is current temp, 2nd number is target per sensor
with redundant temp sensor added to report
 T:190.26 /190.00 R:191.58 /190.00 B:59.98 /60.00 
redundant target is same as temp 0 target
With ADC values enabled
 T:190.26 /190.00 (891.9) B:59.98 /60.00 (1196.0)
 T:190.26 /190.00 (891.9) R:191.58 /190.00 (901.8) B:59.98 /60.00 (1196.0)

Reply:
I have added a define TEMP_SENSOR_REDUNDANT_DEBUGGING if this is commented out, the temperature report is unchanged. If you un-comment it then you get the additional info about the redundant temp sensor.
It is commented out by default so the temp report is unchanged by default.

Reply:
It will be better to only send the report in response to `M105 R`. The extra `R` parameter can just always be available when the redundant temperature sensor exists.